### DISCUSSION TITLE
I customized some icons, but I can't see the colors I set

### DISCUSSION THREAD

**Discussion by hunterHungry** [FlorianWoelki/obsidian-icon-folder#388] [Q&A] [answered]
I customized some icons, but I can't see the colors I set, and only one icon color can be displayed at a time.
![20240301222027](https://github.com/FlorianWoelki/obsidian-iconize/assets/12185027/ad8d7b5d-882a-49a0-b7f7-d522b14445b1)

**Comment by JuanMartinSalinas** [comment]
Having the same problem here ✋ is there a way to change the color of custom icons?

**Comment by JuanMartinSalinas** [comment, marked as answer]
Nvm just found how to do it, SVG files have this attribute called "fill" so you just change that to "currentColor" and it works fine. Since I had a ton of custom icons I made a quick script with JS but it should be pretty easy to do it manually with VSC

**Reply by hunterHungry** [reply]
> Nvm just found how to do it, SVG files have this attribute called "fill" so you just change that to "currentColor" and it works fine. Since I had a ton of custom icons I made a quick script with JS but it should be pretty easy to do it manually with VSC

Thanks for sharing, i'll check later.

**Reply by FlorianWoelki** [reply]
Well, I've decided intentionally not to color these pre-filled icons because sometimes you want to keep the fill of the original icon. But that's cool that you've solved it!
